src/network/networkd-link.h

```diff
@@ -128,6 +128,7 @@ typedef struct Link {
         bool setting_genmode:1;
         bool ipv6_mtu_set:1;
         bool bridge_mdb_configured:1;
+        bool started:1;
```

Comment:
i'd appreciate a quick comment added here that explains when this is set precisely

Reply:
yep sorry, will update again today to address this and the other previous comments